lib/approval.rb

```diff
@@ -0,0 +1,27 @@
+require 'approval_api_client'
+
+class Approval
+  def self.approval_api
+    Thread.current[:approval_api_instance] ||= raw_api
+  end
+
+  def self.call
+    pass_thru_headers
+    yield approval_api
+  rescue ApprovalAPIClient::APIError => e
+    Rails.logger.error("ApprovalApiClient::ApiError #{e.message}")
+    raise Catalog::ApprovalError, e.message
+  end
+
+  private_class_method def self.raw_api
+    ApprovalApiClient.configure do |config|
+      dev_credentials(config)
```

Comment:
@lindgrenj6 
You need to configure the host, scheme here similar to this
https://github.com/ManageIQ/catalog-api/blob/0d08da4460feb05338ebbae12abf1d78bf15c06d/lib/topological_inventory.rb#L21

Which is set here for Approval

https://github.com/ManageIQ/catalog-deploy/blob/0db2e1a885a13a453850fb719e0bdf19db4eb888/api/deploy_template.yaml#L50

Reply:
haha! I had just pushed this up and then you commented. I caught myself forgetting it.